Index: libnfnetlink-1.0.2/include/libnfnetlink/linux_nfnetlink.h
===================================================================
--- libnfnetlink-1.0.2.orig/include/libnfnetlink/linux_nfnetlink.h	2008-06-18 20:36:57.000000000 +0800
+++ libnfnetlink-1.0.2/include/libnfnetlink/linux_nfnetlink.h	2019-08-30 18:04:36.026372012 +0800
@@ -52,6 +52,7 @@
 #define NFNL_SUBSYS_OSF		5
 #define NFNL_SUBSYS_IPSET		6
 #define NFNL_SUBSYS_COUNT		7
+#define NFNL_SUBSYS_FLOWTABLE		13
 
 #ifdef __KERNEL__
 
